examples/webgl_postprocessing_lut.html

```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - postprocessing - lut</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				color: #000;
+				font-family:Monospace;
+				font-size:13px;
+				text-align:center;
+				font-weight: bold;
+
+				background-color: #000;
+				margin: 0;
+			}
+
+			canvas {
+				width: 100vw;
+				height: 100vh;
+				display: block;
+			}
+
+			#info {
+				color:#fff;
+				background-color: rgba( 0, 0, 0, 0.5 );
+				position: relative;
+				margin: 0 auto -2.1em;
+				top: 0;
+				max-width: 550px;
+				padding: 5px;
+			}
+
+			#notes {
+				color:#fff;
+				background-color: rgba( 0, 0, 0, 0.5 );
+				position: absolute;
+				bottom: 0;
+				right: 0;
+				padding: 5px;
+				z-index:100;
+			}
+
+			a { color: red; }
+		</style>
+	</head>
+	<body>
+
+		<div id="info">
+			<a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> - webgl postprocessing lut - a port of <a href="https://webglsamples.org">color-adjust</a>
+		</div>
+
+		<div id="notes">
+			<a href="http://g3d.cs.williams.edu/g3d/data10/index.html#mesh19" target="_blank" rel="noopener">Lee Perry-Smith</a> head
+		</div>
+
+		<canvas></canvas>
+
+		<script src="../build/three.js"></script>
+
+		<script src="js/shaders/CopyShader.js"></script>
+		<script src="js/shaders/LUTShader.js"></script>
+
+		<script src="js/postprocessing/EffectComposer.js"></script>
+		<script src="js/postprocessing/RenderPass.js"></script>
+		<script src="js/postprocessing/ShaderPass.js"></script>
+
+		<script src="js/loaders/GLTFLoader.js"></script>
+		<script src="js/libs/dat.gui.min.js"></script>
+
+		<script>
+
+const renderer = new THREE.WebGLRenderer( { canvas: document.querySelector( 'canvas' ) } );
```

Comment:
Can you please ensure a correct indentation according to the other examples? 

Besides, please use `var` instead of `let` and `const`.

Reply:
Sorry about the indentation. I meant to fix that before I uploaded 😅

I'd prefer to leave them as let and const if possible. This is not part of the library, it's an example. Furthermore the entire rest for JavaScript world has moved on. Using `var` is not only out of date it is nearly universally acknowledged as bad practice. I don't really want to be spreading bad practices.

Reply:
> This is not part of the library, it's an example. Furthermore the entire rest for JavaScript world has moved on. 

That's true but it's important for us to keep the example code consistent. Even if not all aspects of the code base are up to date.

Reply:
Agreed with @Mugen87 on this one. We're overdue another conversation on whether to start using let and const, but that should be discussed elsewhere. 
In the meantime please keep the code style consistent with other examples.